lib/Service/ObjectService.php

```diff
@@ -485,4 +485,23 @@ public function setSchema(int $schema): void
 	{
 		$this->schema = $schema;
 	}
+
+	/**
+	 * Get the audit trail for a specific object
+	 *
+	 * @param int $register The register ID
+	 * @param int $schema The schema ID
+	 * @param string $id The object ID
+	 * @return array The audit trail for the object
+	 */
+	public function getAuditTrail(int $register, int $schema, string $id): array
+	{
+		$filters = [
+			//'register' => $register,
+			//'schema' => $schema,
```

Comment:
Shouldn't we also implement these filters or add a to-do somewhere? 
Or if none of these 2 options^, just remove it all?

Reply:
The problem here is that we should implement them but they don't work and fixing them was out of scope of this branche, So i added a Todo

Reply:
nvm fixed this otherwise